NickvisionMoney.Shared/Models/Account.cs

```diff
@@ -793,8 +792,6 @@ public async Task<bool> AddTransactionAsync(Transaction transaction)
         cmdAddTransaction.Parameters.AddWithValue("$useGroupColor", transaction.UseGroupColor);
         cmdAddTransaction.Parameters.AddWithValue("$notes", transaction.Notes);
         cmdAddTransaction.Parameters.AddWithValue("$tags", string.Join(',', transaction.Tags));
-        Tags.AddRange(transaction.Tags.Where(t => !Tags.Contains(t)));
```

Comment:
Why was this removed? Shouldn't it be kept if new tags are added to a transaction that don't exist in an account?

Reply:
We do this in controller, and by doing this we ensure that a new TagButton gets added in the UI